web/client/themes/default/less/leaflet.less

```diff
@@ -71,11 +71,14 @@
 }
 
 .leaflet-control-scale {
-    top: 10px;
-    right: 10px;
+    bottom: 5px;
+    right: 45px ;
```

Comment:
Now we could apply a right value of 55px so widget tray and scale bar are aligned